src/banner/toast.component.ts

```diff
@@ -28,7 +28,7 @@ import { Banner } from "./banner.component";
 			(click)="onClose()">
 			<svg
 				class="bx--toast-notification-icon"
-				aria-label="close"
+				[attr.aria-label]="bannerObj.closeLabel"
```

Comment:
this label should be on the button to fix DAP violation: `Form controls must have associated labels`. See #194 for more clarification